### PR TITLE
Add Hello World DSA

### LINKED ISSUE #837: [Add] Hello World DSA
### Site URL

https://www.youtube.com/@HelloWorldbyprince/playlists

### Description of the above provided site

Hello World youtube channels provides C++ specific DSA Course focused on explanation of Algorithms.  

### Category

Youtube

### Sub Category

DSA

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Vidip-Ghosh**
## Fixes Issue
Closes #837 

## Changes proposed

Added Hello World DSA playlist.

### CODE REVIEW

**Review by navharsh — CHANGES_REQUESTED** [review]
You should remove firebase commit of PR #849 added here as well.

**Comment by Vidip-Ghosh** [comment]
> You should remove firebase commit of PR #849 added here as well.

Yes, I have removed.

**Review by navharsh — DISMISSED** [review]
LGTM!

File: database/youtube/data-structures.json [hunk 1]
```diff
@@ -94,5 +94,13 @@
     "category": "youtube",
     "subcategory": "dsa",
     "language": "english"
+  },
+  {
+    "name":"Hello World DSA", 
+    "description":"Hello World youtube channels provides C++ specific DSA Course focused on explanation of Algorithms.",
```
Comment (CBID2) [inline database/youtube/data-structures.json:100]: [Proofreading]
```suggestion
    "description": "This channel provides C++ specific DSA content. This course also provides explanations of Algorithms.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Vidip-Ghosh! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Other than that, you're off to a great start! :)

**Comment by Vidip-Ghosh** [comment]
> Hi @Vidip-Ghosh! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Other than that, you're off to a great start! :)

Yes I made changes

**Review by CBID2 — DISMISSED** [review]
LGTM! :)

**Review by rupali-codes — APPROVED** [review]

**Review by CBID2 — APPROVED** [review]
LGTM!

**Comment by CBID2** [comment]
@rupali-codes and @ujjawaltyagii, for some reason, this PR is missing successful preview deployment.

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**